### PR TITLE
Made aria-multiline true by default in RichText

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
There was a misunderstanding of what aria-multiline means. Currently, Gutenberg only used area-multiline true when multi lines could be inserted by pressing enter e.g.: the quote value and the caption of the image. Aria-multiline was set to false when RichText inputs could not have a new line inserted by pressing enter.

aria-multiline should be true in all the inputs with can have multiple lines rendered on the screen (behave like a text area) even if these lines were not created because of line breaks but because of line wrapping (not having space on display to show them in a single line). According to this interpretation, it looks like all the Gutenberg RichgText usage should use aria-multiline = true because all of our RichText use cases are more close to TextAreas to then single line input fields and they may have multiple lines rendered on a screen.

aria-multiline is now true by default in all RichText components.

## How has this been tested?
I verified all RichText usages render aria-multiline=true in their dom nodes.

### CODE REVIEW

File: editor/components/rich-text/index.js [hunk 1]
```diff
@@ -852,9 +852,10 @@ export class RichText extends Component {
 			formatters,
 			autocompleters,
 			format,
+			ariaMultiline = true,
```
Comment (gziolo) [inline editor/components/rich-text/index.js]: Let's document this prop before it is merged.

**Review by gziolo — APPROVED** [review]
LGTM, we should include this new prop in README file before we proceed.

File: editor/components/rich-text/index.js (hunk 1 above)
Comment (jorgefilipecosta) [reply editor/components/rich-text/index.js]: Documentation was added. Besides documenting I did another update to simplify the code. Now we use the same name as the default aria prop "aria-multiline" but we make it default to true so most RichText users don't need to worry about setting the prop.

**Review by gziolo — APPROVED** [review]
Thanks 🙇

**Comment by afercia** [comment]
Thanks @jorgefilipecosta and everyone. Late to the party, just to confirm blocks editable areas are now announced as textareas and that's what we want to communicate to assistive technology users 🙂 

Before:

screen shot 2018-06-18 at 18 40 29

After:

screen shot 2018-06-18 at 18 42 01

**Pull request merged**